internal/config/attest.go

```diff
@@ -3,8 +3,8 @@ package config
 import "github.com/spf13/viper"
 
 type attest struct {
-	Key      string `yaml:"key" json:"key" mapstructure:"key"`
-	Password string `yaml:"password" json:"password" mapstructure:"password"`
+	Key      string `yaml:"-" json:"-" mapstructure:"key"`
+	Password string `yaml:"-" json:"-" mapstructure:"password"`
```

Comment:
nit: mind adding comments similar to https://github.com/anchore/syft/blob/main/internal/config/registry.go#L13 ?